packages/perseus/src/widgets/interactive-graphs/axis-ticks.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from "react";
+
+import {useTransform} from "./graphs/use-transform";
+
+import type {vec} from "mafs";
+
+const tickSize = 10;
+
+const tickStyle: React.CSSProperties = {
+    stroke: "black",
+    strokeWidth: 1,
+};
+
+const YGridTick = ({y}: {y: number}) => {
+    const startingPoint: vec.Vector2 = [0, y];
+    let [startPtPx] = useTransform(startingPoint);
+
+    const endPtPx = [startPtPx[0] + tickSize / 2, startPtPx[1]];
+    startPtPx = [startPtPx[0] - tickSize / 2, startPtPx[1]];
+
+    return (
+        <g className="y-axis-ticks" tabIndex={0} style={{} as any}>
+            <line
+                x1={startPtPx[0]}
+                y1={startPtPx[1]}
+                x2={endPtPx[0]}
+                y2={endPtPx[1]}
+                style={tickStyle}
+            />
+        </g>
+    );
+};
+
+const XGridTick = ({x}: {x: number}) => {
+    const startingPoint: vec.Vector2 = [x, 0];
+    let [startPtPx] = useTransform(startingPoint);
+
+    const endPtPx = [startPtPx[0], startPtPx[1] + tickSize / 2];
+    startPtPx = [startPtPx[0], startPtPx[1] - tickSize / 2];
+
+    return (
+        <g className="x-axis-ticks" tabIndex={0} style={{} as any}>
```

Comment:
Why pass an empty `style` object here? (and in `YGridTick` above?)

And do we really want these groups to be tabbable?

Reply:
I believe this snuck in here as we copied the "g" element from elsewhere! I'll remove them :D